docs/tempo/website/troubleshooting/unable-to-see-trace.md

```diff
@@ -52,7 +57,36 @@ This can also be confirmed by checking the metric `tempo_request_duration_second
 - Check logs of distributors for a message like `msg="pusher failed to consume trace data" err="DoBatch: IngesterCount <= 0"`.
   This is likely because no ingester is joining the gossip ring, make sure the same gossip ring address is supplied to the distributors and ingesters.
 
-## Section 2: Diagnosing and fixing issues with querying traces
+## Section 2: Diagnosing and fixing sampling & limits issues
+
+If you are able to query some traces in Tempo but not others, you have come to the right section!
+
+This could happen because of a number of reasons and some have been detailed in this blog post -
+[Where did all my spans go? A guide to diagnosing dropped spans in Jaeger distributed tracing](https://grafana.com/blog/2020/07/09/where-did-all-my-spans-go-a-guide-to-diagnosing-dropped-spans-in-jaeger-distributed-tracing/).
+This is useful if you are using the Jaeger Agent.
+
+If you are using the Grafana Agent, continue reading the following section for metrics to monitor.
+
+### Diagnosing the issue
+Check if the pipeline is dropping spans. The following metrics on the Grafana Agent help determine this -
+- `tempo_exporter_send_failed_spans`. The value of this metric should be 0.
+- `tempo_receiver_refused_spans`. This value of this metric should be 0.
+- `tempo_processor_dropped_spans`. The value of this metric should be 0.
+
+If the pipeline is not reporting any dropped spans, check whether application spans are being dropped by Tempo. The following metrics help determine this -
```

Comment:
I think this is not very clear. `tempo_receiver_refused_spans` represents the number of spans dropped by Tempo and the Grafana Agent. I think it's worth explaining that both share the same metric.